templates/edge-computing/index.html

```diff
@@ -10,468 +13,457 @@
   https://docs.google.com/document/d/1DdOhFZyvaew0NWW2Uo-EtaB81wzCy-ApdRmanTinyVM/edit#
 {% endblock meta_copydoc %}
 
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
 {% block content %}
 
-  <section class="p-strip--suru-topped">
-    <div class="row u-equal-height">
-      <div class="col-7 u-vertically-center">
-        <h1>Micro clouds</h1>
-        <p class="p-heading--4">Edge computing solution</p>
-        <p>Serving compute near the consumers, reusing cloud APIs.</p>
-        <p>
-          <a href="/engage/guide-to-micro-clouds"
-             class="p-button--positive u-no-margin--bottom">Download the whitepaper</a>
-        </p>
-      </div>
-      <div class="col-5 u-align--center u-vertically-center u-hide--medium u-hide--small">
-        {{ image(url="https://assets.ubuntu.com/v1/f24913d5-Edge+Cloud.svg",
-                alt="Micro clouds for edge computing",
-                width="365",
-                height="250",
+  {% call(slot) vf_hero(
+    title_text='Micro clouds',
+    subtitle_text='Edge computing solution',
+    layout='50/50-full-width-image'
+    ) -%}
+    {%- if slot == 'description' -%}
+      <p>
+        Serving compute near the consumers, reusing cloud APIs. Compose unique edge solutions using the best open source components, anywhere at scale.
+      </p>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <a href="/engage/guide-to-micro-clouds" class="p-button--positive">Get the whitepaper</a>
+    {%- endif -%}
+    {%- if slot == 'image' -%}
+      <div class="p-image-container--cinematic is-cover">
+        {{ image(url="https://assets.ubuntu.com/v1/3909b910-hero.png",
+                alt="",
+                width="2464",
+                height="1027",
                 hi_def=True,
-                loading="auto") | safe
+                loading="auto",
+                attrs={"class": "p-image-container__image"}) | safe
         }}
       </div>
-    </div>
-  </section>
+    {% endif -%}
+  {% endcall -%}
 
-  <section class="p-strip--suru">
-    <div class="u-fixed-width">
-      <p class="p-heading--2">Compose unique edge solutions using the best open source components, anywhere at scale.</p>
+  <section class="p-section">
+    <hr class="is-fixed-width" />
+    <div class="row--50-50">
+      <div class="col">
```

Comment:
```suggestion
    <div class="row--50-50">
        <hr class="p-rule" />
      <div class="col">
```

Reply:
Sorry I didn't get it, what do you want me to change here?